cli/tests/upstream.rs

```diff
@@ -100,6 +113,20 @@ async fn upstream_sync() -> TestResult {
         "/hello/greeting.html"
     );
 
+    ////////////////////////////////////////////////////////////////////////////////////
+    // Test that override_host works as intended
+    ////////////////////////////////////////////////////////////////////////////////////
+
+    let resp = test
+        .against(
```

Comment:
```suggestion
    let resp = test
        .via_hyper()
        .against(
```

Can we use `Test::via_hyper` here, so we're also sending this request through Hyper? It ought not to affect the result, but it'd bring some additional peace of mind that we won't see any regressions :slightly_smiling_face: